### ISSUE TITLE
Dev Tees - Headphones shirt

### ISSUE THREAD

**Issue by adamculpepper** [jglovier/ama#11]
Hey Joel, I sent you an email several months ago asking if you planned on doing another run of the pixelated headphones shirt again. That was my absolute favorite shirt and I lost it in 2016 when my city flooded. Obviously I lost everything (including this shirt), but I still think about it today.

**Comment by jglovier** [comment]
Hey Adam! I'm so sorry to hear you lost everything in a flood. 😞💔 I'm also really sorry I never responded to your email! I took a new job in the spring and haven't logged into that account all year. But thanks for taking the time to reach me here.

The answer is I think I completely forgot about that shirt in transitioning my catalog of designs over to Amazon Merch. So thanks for speaking up about your interest in it! When I get a few minutes I'll post the shirt there and let you know when it's available again.

**Comment by adamculpepper** [comment]
I figured something was up and have been checking every year or so to see if it's back in your shop again. That sounds great, thanks! ❤️

**Comment by jglovier** [comment]
OMG it seems bonkers that I wasn't able to get that shirt up on the store in the past 6 months! I got halfway there a few months ago by converting the artwork from the old system I was using to be formatted to the new system, but just never got a chance to upload it and create the product listing.

However...I found some time yesterday (thanks for the ping here btw!) and submitted the product listing, and [it is live now](https://amzn.to/3vJnTzH)!

@adamculpepper I'd love to just buy the shirt for you and have it sent to you out of appreciation for your long-time support of Dev Tees, and out of compassion for the situation that resulted in your losing so much of your things. If you DM me your shipping address on Twitter I'll order it and have it shipped to you! ❤️ 📦 👕
